requirements.txt

```diff
@@ -1,3 +1,4 @@
 ovos-plugin-manager>=0.0.1
+ovos-bus-client>=0.0.6a22
 ovos_utils
 mycroft-messagebus-client
```

Comment:
mycroft-messagebus-client should be deleted

Reply:
should ovos-utils be versioned at `>=0.0.38`?

Reply:
technically the package allows lower versions of utils, but bus client sound be 0.0.8 to have the new imports available

Reply:
I did get the version wrong, but I don't think it needs to be 0.0.8

The gui api was introduced in 0.0.6a23
https://github.com/OpenVoiceOS/ovos-bus-client/blob/ef6e47622971b2844df74bdb6344e8f18c11e2fc/ovos_bus_client/apis/gui.py#L66

Should we not stick with the lowest working version?

Reply:
what matters is the last stable bus client that introduced the new import, so you are correct should be 0.0.6 not 0.0.8, i didnt check and got the version the wrong :) 

in this case, bus client itself will require a higher utils, so ovos-utils version is irrelevant... good practice to keep the lowest working version pinned however.